snap/snapcraft.yaml

```diff
@@ -0,0 +1,42 @@
+name: onedrive # you probably want to 'snapcraft register <name>'
+base: core20 # the base snap is the execution environment for this snap
+version: '0.1' # just for humans, typically '1.2+git' or '1.3.2'
```

Comment:
Should this not be the version of the application?

Reply:
`2.4.3-snap1` or so I would suggest